cfgov/jinja2/v1/_includes/molecules/related-links.html

```diff
@@ -19,16 +19,23 @@
    ========================================================================== #}
 
 <div class="m-related-links">
-    <h2 class="header-slug">
-        <span class="header-slug_inner">
-            {{ value.heading }}
-        </span>
-    </h2>
-    <ul class="list list__unstyled list__links">
-    {% for link in value.links %}
-        <li class="list_item">
-            <a href="{{ link.url }}" class="list_link">{{ link.text }}</a>
-        </li>
-    {% endfor %}
-    </ul>
+    {% if value.heading %}
+        <h2 class="header-slug">
+            <span class="header-slug_inner">
+                {{ value.heading }}
+            </span>
+        </h2>
+    {% endif %}
+
+    {{ value.paragraph }}
```

Comment:
@sebworks shouldn't matter since if paragraph is `None`, nothing will show up here